book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accesor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] Kindly add the method for associating with the Person and Rental classes.

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```

Reply:
- Well resolved @Ibtisam34